oak-lucene/pom.xml

```diff
@@ -107,7 +107,7 @@
             <_exportcontents>
               !org.apache.lucene.queryparser.xml,
               !org.apache.lucene.queryparser.xml.builders,
-              org.apache.lucene.*;version=${lucene.version}-oak1
+              org.apache.lucene.*;version=${lucene.version}-oak2
```

Comment:
What is `oak1` and `oak2`?

Reply:
lucene.version is 4.7.2. We could call all of this 4.7.3, but I assume the Lucene project would object. Thus the "oak" qualifier that should make clear that this is "our" version. (oak1 is the copy of 4.7.2 with minimal changes for JDK 11 compliance - see https://github.com/apache/jackrabbit-oak/pull/1439/commits/bca1e9c63c470967e3bee363d22c2026219d49e8 - "oak2" is this fix).